extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/JPAConfig.java

```diff
@@ -38,8 +42,35 @@ public JPAConfig(JPAConfigSupport jpaConfigSupport) {
     }
 
     void startAll() {
+        List<CompletableFuture<?>> start = new ArrayList<>();
```

Comment:
@yrodiere @Sanne could one of you have a look at this part? It changes how persistence units are started.

Reply:
LGTM. The `LazyPersistenceUnit` is apparently thread-safe. I cannot understand why it exists, since it's started eagerly anyway, but that's another matter.